Gemfile

```
@@ -5,8 +5,7 @@ gemspec
 
 # Private runtime dependencies
 gem 'nokogiri', '~> 1.13', force_ruby_platform: true
-gem 'sisjwt', '~> 0.2.4',
-    github: 'tractionguest/sisjwt-ruby'
+gem 'sisjwt', github: 'tractionguest/sisjwt-ruby', ref: 'e5e585a'
```

change to
```suggestion
gem 'sisjwt', '~> 0.3.0'
```
when this [PR](https://github.com/tractionguest/sisjwt-ruby/pull/7) is merged